core/postfix/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM alpine:3.8
 
 RUN apk add --no-cache postfix postfix-pcre rsyslog \
  && python3 py3-pip \
```

Comment:
Omit `&&` beginning of line

Reply:
Basically, your first typo rectification was wrong itself. The Dockerfile was correct with the merge you did:
https://github.com/Mailu/Mailu/blob/00b5ae11db3f41d81c0b64b39bdddd99ce5a73a9/core/dovecot/Dockerfile

The Travis build for the merge commit failed only on running of rspamd, which is a known issue (#627).
https://travis-ci.org/Mailu/Mailu/builds/439491439#L1373
I've now re-triggered the build for that specific commit.

I think if you revert your last two commits in this branch (the typo's), we're good to go.